CMakeLists.txt

```diff
@@ -36,6 +36,7 @@ endif()
 #-----------------------------------------------------------------------------
 # Extension modules
 add_subdirectory(Dcm2niixGUI)
+add_subdirectory(Dcm2niixPlugin)
```

Comment:
Might be better named DICOMDcm2niixPlugin for clarity (and consistency [e.g. with QR](https://github.com/QIICR/QuantitativeReporting/tree/master/DICOMPlugins))